spring-integration/spring-boot-integration-test/src/main/java/ai/timefold/solver/spring/boot/it/TimefoldController.java

```diff
@@ -0,0 +1,25 @@
+package ai.timefold.solver.spring.boot.it;
+
+import ai.timefold.solver.core.api.solver.SolverFactory;
+import ai.timefold.solver.spring.boot.it.domain.IntegrationTestSolution;
+
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/integration-test")
+public class TimefoldController {
```

Comment:
Timefold is a company; we have Solver, we have Orbit. Let's start using `TimefoldSolver` consistently, because we're dealing with the solver here, not with the company.